proto/cosmos/base/v1beta1/coin.proto

```diff
@@ -17,7 +18,7 @@ message Coin {
 
   string denom  = 1;
   string amount = 2
-      [(cosmos_proto.scalar) = "cosmos.Int", (gogoproto.customtype) = "Int", (gogoproto.nullable) = false];
+      [(cosmos_proto.scalar) = "cosmos.Int", (gogoproto.customtype) = "Int", (gogoproto.nullable) = false, (cosmos.msg.legacy_amino.v1.dont_omitempty) = true];
```

Comment:
(golang-only): This is annoying. The correct way is to add `(cosmos.msg.legacy_amino.v1.dont_omitempty) = true`, because of nullable=false.

But when generating the go code, this goes into `types/coin.pb.go`, with a dep cycle with `types/msgservice`.

One solution is to generate the msg.legacy_amino.v1 stuff into a new subpackage, maybe `types/tx/legacy_amino`?

Reply:
Proto packages should always generate to a unique go package